store/src/java/com/zimbra/qa/unittest/TestMailItem.java

```diff
@@ -34,29 +40,51 @@
 import com.zimbra.cs.mailbox.MailboxManager;
 import com.zimbra.cs.service.formatter.HeadersOnlyInputStream;
 
-public class TestMailItem extends TestCase {
+public class TestMailItem {
 
-    private static final String USER_NAME = "user1";
+    @Rule
+    public TestName testInfo = new TestName();
+    private Account account = null;
     private static final int TEST_CONTACT_ID = 9999;
+    private static final Provisioning prov = Provisioning.getInstance();
 
-    @Override
+    @Before
     public void setUp() throws Exception {
```

Comment:
not sure what is the point of an empty setUp() method. Shouldn't cleanUp be called here?